als_driver.c

```diff
@@ -0,0 +1,22 @@
+#define ALS_EN_GPIO 11
+#define ALS_READ_ADDR 0xA110FF30
+
+
+#include <stdio.h>
+#include <stdint.h>
+
+int als_read_reg(uint32_t als_reg_addr)
+{
+    printf("read value of 0x%X from als at addr 0x%X\n",
+           200, ALS_READ_ADDR);
```

Comment:
what's with the magic number here?

Reply:
Oh yeah, I need to fix that...

Reply:
Ok, this fix looks good.

Reply:
oh wait, shouldn't this actually be reading the h/w, not faking out a constant value?

Reply:
Wouldn't it be nice if I could resolve this comment....